src/lib/DataConnectStore.js

```diff
@@ -59,6 +59,22 @@ export default class DataConnectStore {
     return useCase.connectors.map(c1 => this.find(c2 => c1.slug === c2.slug))
   }
 
+  // Fetch all accounts and updates their matching connectors
+  fetchAllAccounts (index) {
+    if (!index && this.accountsIndex) index = this.accountsIndex
```

Comment:
In my opinion, the store does not even need to know that we need an index, it's a couchdb implementation concern, not ours. So IMHO we should deal with the index directly into the `accounts` lib.

An example is already done with the `findBySlug` method for the konnectors : https://github.com/cozy/cozy-data-connect/blob/master/src/lib/konnectors.js#L29
As you call this method from outside, you can never tell that there is an index involved.